### PR TITLE
added info to substep

### PR DESCRIPTION
**Pull request by jc-berger**
See Jira: https://issues.redhat.com/browse/ACM-8661

### CODE REVIEW

File: clusters/cluster_lifecycle/cim_create_cli.adoc [hunk 1]
```diff
@@ -117,14 +117,14 @@ spec:
...
-<16> The Red Hat CoreOS image version to use for your `InfraEnv`. Make sure the version refers to the OS image specified in the `AgentServiceConfig.spec.osImages`, and that `OSImageVersion` matches an {ocp-short} version in the OS images list. You cannot specify `OSImageVersion` and `ClusterRef` at the same time. See _Enabling the central infrastructure management service_ for more information.
+<16> The Red Hat CoreOS image version to use for your `InfraEnv`. Make sure the version refers to the OS image specified in the `AgentServiceConfig.spec.osImages`, and that `OSImageVersion` matches an {ocp-short} version in the OS images list. You cannot specify `OSImageVersion` and `ClusterRef` at the same time. By default, each release has a specific set of Red Hat CoreOS image versions that are relevant to the next release. Therefore, if you try to add another version, and the Red Hat CoreOS image version does not exist by default, then you must manually add the version. To learn more about adding versions, see _Enabling the central infrastructure management service_.
```
Comment (CrystalChun) [inline clusters/cluster_lifecycle/cim_create_cli.adoc]: I think we want to let the user know that they need to use a version that either they specify in AgentServiceConfig or one that exists by default. Attempted to reword to say that: 

```suggestion
<16> The Red Hat CoreOS image version to use for your `InfraEnv`. Make sure the version refers to the OS image specified in either the `AgentServiceConfig.spec.osImages` or in the default OS images list. Each release has a specific set of Red Hat CoreOS image versions. The `OSImageVersion` must match an {ocp-short} version in the OS images list. You cannot specify `OSImageVersion` and `ClusterRef` at the same time. If you want to use another version of the Red Hat CoreOS image that does not exist by default, then you must manually add the version by specifying it in the `AgentServiceConfig.spec.osImages`. To learn more about adding versions, see _Enabling the central infrastructure management service_. 
```

Feel free to reword!

**Review by dockerymick — APPROVED** [review]
Reviewed this and mainly focused on what you updated. Those changes look good, but I see future tense being used throughout the file. Will you help with updating that?

**Review by CrystalChun — APPROVED** [review]
/lgtm

**Review by dockerymick — APPROVED** [review]
@jc-berger thanks for the updates and accepting my suggestions! This looks good to me

**Pull request merged**